### PR TITLE
[RFR] Bootstrap the project

### PR DESCRIPTION
**Pull request by Luwangel**
- [x] Create the readme
- [x] Build an initial setup with unit tests
- [x] Create a dockerfile

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# https://travis-ci.com/marmelab/15-puzzle-go
+
+language: go
+
+go:
+  - 1.7.3
```
Comment (Kmaschta) [inline .travis.yml]: Can you use the latest version of Go ?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,6 @@
+FROM golang:1.8
```
Comment (Kmaschta) [inline Dockerfile]: Please use the latest version

File: README.md [hunk 3]
```diff
@@ -1,2 +1,45 @@
 # 15-puzzle-go
-A CLI tool to play the 15-puzzle game, in Go
+
+A CLI tool to play the 15-puzzle game, in Go.
+
+## Contributing
+
+### Help
+
+Print all commands available
```
Comment (Kmaschta) [inline README.md]: s/all commands available/all available commands/

**Review by Kmaschta — CHANGES_REQUESTED** [review]

File: Dockerfile [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+
+ENV GOPATH /src
+
+VOLUME /src
```
Comment (jpetitcolas) [inline Dockerfile]: Useless. Remove to keep it simple.

File: Makefile [hunk 5]
```diff
@@ -0,0 +1,17 @@
+PWD = $(shell pwd)
```
Comment (jpetitcolas) [inline Makefile]: There is already a environment variable `${PWD}`.

**Review by jpetitcolas — CHANGES_REQUESTED** [review]

**Review by Kmaschta — APPROVED** [review]

**Pull request merged**